### DISCUSSION TITLE
<Leader+F> how to not show node_modules/target folders content?

### DISCUSSION THREAD

**Discussion by oneslash** [helix-editor/helix#7941] [Q&A] [answered]
Hi,

When I open the file picker, I see all the files which I do not need to edit such as files in folders:

- node_modules (node)
- vendor (go)
- target (rust)
- etc.

How do I ignore them?

**Comment by pascalkuthe** [comment, marked as answer]
helix respects `gitignore` and `ignore` files. These kind of build artifacts are usually ignored anyway.

You can also manually ignore directories with negative matches `!vendor` to hide the vendor directory for example

**Reply by oneslash** [reply]
Thank you 🙏

**Reply by gsrec** [reply]
It doesn't seem to work for version 25.07.1. In mine `.gitignore` has the string `node_modules` (I've tried various combinations, with and without glob patterns), default settings (I've also tried various settings of `editor.file-picker`), it does not work for nested directories (if a directory with several `nodejs` projects is open), and it does not work when opening a current project (at the root of `.gitignore`, `node_modules`). However, the `.ignore` file works fine and recursively. PS You are creating the best code editor.

**Reply by PlethoraChutney** [reply]
Same issue with .gitignore not working in 25.07.1, but .ignore does work

**Reply by klukaszek** [reply]
Bump. Same issue on 25.07.

`.ignore` files work, but `.gitignore` don't seem to work.

Tested with no direct configuration, as well as:
```
[editor.file-picker]
git-ignore = true
```
